### PR TITLE
Daemon: use msg queue and specific task for handling status messages

### PR DESCRIPTION
**Pull request by emlowe**
Use a message queue and a tracked task for sending messages off the queue. This should ensure FIFO sending of the messages and proper cleanup of tasks.

previous code would create tasks to send each status message, however, this could result in messages on the wire in a different order, as the tasks are not guaranteed to execute in FIFO order. Additionally, these tasks were not tracked or managed, so other types of strange issues could arise.

Also combine "keyring" status messages with status messages into a single queue, to eliminate some copy-paste sending code.

Added some tests specifically for testing keyring status messages

### CODE REVIEW

File: chia/daemon/server.py [hunk 1]
```diff
@@ -144,6 +145,18 @@ def _get_keys_by_fingerprints(fingerprints: Optional[List[uint32]]) -> Tuple[Lis
     return keys, missing_fingerprints
 
 
+@dataclasses.dataclass(frozen=True)
+class StatusMessage:
+    service: str
+    command: str
+    destination: str
+    origin: str
```
Comment (altendky) [inline chia/daemon/server.py:153]: Should these be more clearly constrained than just `str`?  An enumeration?  I know the remote entities service id is just any string they want but I think some of these are internal here.

File: tests/core/daemon/test_daemon.py [hunk 2]
```diff
@@ -51,6 +51,20 @@ def id(self) -> str:
         return f"{self.route}: {self.description}"
 
 
+@dataclass
+class RouteCaseStatus:
```
Comment (altendky) [inline tests/core/daemon/test_daemon.py:55]: I expect `Case` to be at the end, but perhaps I'm misreading this somehow.

**Review by altendky — CHANGES_REQUESTED** [review]

File: chia/daemon/server.py (hunk 1 above)
Comment (emlowe) [reply chia/daemon/server.py:153]: command could be constrained to an enum as there are a known fixed number of supported commands. it's currently defined in a dict in `get_command_mapping`

File: chia/daemon/server.py [hunk 3]
```diff
@@ -737,29 +727,43 @@ def extract_plot_queue(self, id=None) -> List[Dict]:
...
-    async def _state_changed(self, service: str, message: Dict[str, Any]):
-        """If id is None, send the whole state queue"""
-        if service not in self.connections:
-            return None
-
-        websockets = self.connections[service]
+    async def _process_state_changed_queue(self) -> None:
+        while True:
+            message = await self.state_changed_msg_queue.get()
+            await self._state_changed(message)
```
[the remark this answers is not in the document]
Comment (emlowe) [reply chia/daemon/server.py]: Good idea - added in the same as ping task.

File: tests/core/daemon/test_daemon.py (hunk 2 above)
Comment (emlowe) [reply tests/core/daemon/test_daemon.py]: adjusted

File: chia/daemon/server.py (hunk 1 above)
Comment (emlowe) [reply chia/daemon/server.py:153]: I think it's just easier to leave this as is.

**Review by altendky — APPROVED** [review]

**Review by wjblanke — APPROVED** [review]
aok

**Pull request merged**